From 44fa2fcf687f76acbadf066746df5cfa97d39074 Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@freescale.com>
Date: Fri, 6 Oct 2017 12:16:42 +0300
Subject: [PATCH 061/706] arm64: dts: update ENETC node for BG0.9 ver 62

Applicable for simulator model version m0165 and above,
to match ENETC BG0.9 ver 62 specification (and newer versions).

Signed-off-by: Alex Marginean <alexandru.marginean@freescale.com>
(cherry picked from commit eed6dd9f01bf06ef55c255b128679f2077cc900e)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 29 +++++--------------
 1 file changed, 8 insertions(+), 21 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 0c432095d9a7..693f9ce3f274 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -481,27 +481,14 @@
 			msi-parent = <&its>;
 			device_type = "pci";
 			bus-range = <0x0 0x0>;
-			ranges = <0x82000000 0x0 0x00000000  0x1 0xff000000  0x0 0x100000     /* non-prefetchable memory */
-				  0xc2000000 0x0 0x00000000  0x1 0xff100000  0x0 0x100000     /* prefetchable memory */
-				  0x82000000 0x0 0x00000000  0x1 0xff200000  0x0 0x100000     /* non-prefetchable memory */
-				  0xc2000000 0x0 0x00000000  0x1 0xff300000  0x0 0x100000     /* prefetchable memory */
-
-				  0x82000000 0x0 0x00000000  0x1 0xfe000000  0x0 0x100000     /* non-prefetchable memory */
-				  0xc2000000 0x0 0x00000000  0x1 0xfe100000  0x0 0x100000     /* prefetchable memory */
-				  0x82000000 0x0 0x00000000  0x1 0xfe200000  0x0 0x100000     /* non-prefetchable memory */
-				  0xc2000000 0x0 0x00000000  0x1 0xfe300000  0x0 0x100000     /* prefetchable memory */
-
-				  0x82000000 0x0 0x00000000  0x1 0xfd000000  0x0 0x100000     /* non-prefetchable memory */
-				  0xc2000000 0x0 0x00000000  0x1 0xfd100000  0x0 0x100000     /* prefetchable memory */
-				  0x82000000 0x0 0x00000000  0x1 0xfd200000  0x0 0x400000     /* non-prefetchable memory */
-
-				  0x82000000 0x0 0x00000000  0x1 0xfc000000  0x0 0x100000     /* non-prefetchable memory */
-				  0xc2000000 0x0 0x00000000  0x1 0xfc100000  0x0 0x100000     /* prefetchable memory */
-
-				  0x82000000 0x0 0x00000000  0x1 0xfb000000  0x0 0x100000     /* non-prefetchable memory */
-				  0xc2000000 0x0 0x00000000  0x1 0xfb100000  0x0 0x100000     /* prefetchable memory */
-
-				  0x82000000 0x0 0x00000000  0x1 0xfa000000  0x0 0x100000>;   /* non-prefetchable memory */
+			ranges = <0x82000000 0x0 0x00000000  0x1 0xf8000000  0x0 0x0C0000     /* PF0-2, BAR0 - non-prefetchable memory */
+				  0x82000000 0x0 0x00000000  0x1 0xf8100000  0x0 0x060000     /* PF3-5, BAR0 - non-prefetchable memory */
+				  0xc2000000 0x0 0x00000000  0x1 0xf8160000  0x0 0x050000     /* PF0-5, BAR2 - prefetchable memory */
+				  0x82000000 0x0 0x00000000  0x1 0xf81c0000  0x0 0x020000     /* PF0, VF-BAR0 - non-prefetchable memory */
+				  0xc2000000 0x0 0x00000000  0x1 0xf81e0000  0x0 0x020000     /* PF0, VF-BAR2 - prefetchable memory */
+				  0x82000000 0x0 0x00000000  0x1 0xf8200000  0x0 0x020000     /* PF1, VF-BAR0 - non-prefetchable memory */
+				  0xc2000000 0x0 0x00000000  0x1 0xf8220000  0x0 0x020000     /* PF1, VF-BAR1 - prefetchable memory */
+				  0x82000000 0x0 0x00000000  0x1 0xfc000000  0x0 0x400000>;   /* PF2/PF5, BAR4 - non-prefetchable memory */
 		};
 	};
 };
-- 
2.17.1

